openbsd/OpenBSDProcessList.h

```diff
@@ -43,7 +43,7 @@ typedef struct OpenBSDProcessList_ {
 
    CPUData* cpus;
    int cpuSpeed;
-
+   int *cpuIndex;
```

Comment:
What is the point of this array?
It is only written to, but never read from.

Reply:
Good point, it's a useless remnant of an earlier attempt at skipping offline cpus, I've pushed a commit to drop that.